src/app/instances/view.tsx

```diff
@@ -56,7 +56,8 @@ function InstanceView({
   const report = api.job.report.useQuery(
     { jobId },
     {
-      refetchInterval: 5000,
+      refetchInterval: (q) =>
+        q.state.data?.state === "RUNNING" ? 1000 * 60 * 2 : false,
```

Comment:
We need to do the same thing for the "PENDING" status, since the job does not start immediately.

Reply:
done.